### PR TITLE
[zh-cn] sync kubectl/_index kubectl quick-reference reference/_index

### PR DESCRIPTION
**Pull request by my-git9**
content/zh-cn/docs/reference/_index.md
content/zh-cn/docs/reference/kubectl/quick-reference.md
content/zh-cn/docs/reference/kubectl/_index.md
content/zh-cn/docs/reference/kubectl/quick-reference.md

### CODE REVIEW

File: content/zh-cn/docs/reference/kubectl/quick-reference.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ These instructions are for Kubernetes v{{< skew currentVersion >}}. To check the
 
 ### BASH
 -->
-## Kubectl 自动补全   {#kubectl-autocomplete}
+## kubectl 自动补全   {#kubectl-autocomplete}
```
Comment (tico88612) [inline content/zh-cn/docs/reference/kubectl/quick-reference.md:43]: I'd like to know why some of the titles of this document start with capital letters and some with lowercase letters.

File: content/zh-cn/docs/reference/kubectl/kubectl.md [hunk 2]
```diff
@@ -543,6 +555,20 @@ When set to true, external plugins can be used as subcommands for builtin comman
...
+<tr>
+<td colspan="2">KUBECTL_PORT_FORWARD_WEBSOCKETS</td>
+</tr>
+<tr>
+<td></td><td style="line-height: 130%; word-wrap: break-word;">
+<!--
+When set to true, the kubectl port-forward command will attempt to stream using the websockets protocol.
+If the upgrade to websockets fails, the commands will fallback to use the current SPDY protocol.
+-->
+当设置为 true 时，`kubectl port-forward` 命令将尝试使用 websockets 协议进行流式传输。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubectl/kubectl.md:567]:
```suggestion
当设置为 true 时，`kubectl port-forward` 命令将尝试使用 WebSocket 协议进行流式传输。
```

File: content/zh-cn/docs/reference/kubectl/quick-reference.md (hunk 1 above)
Comment (my-git9) [reply content/zh-cn/docs/reference/kubectl/quick-reference.md:43]: There is no standard in Chinese for capitalizing the first letter of a sentence.
And `kubectl` is a fixed word for Kubernetes, let alone capitalizing the first letter.

Comment (windsonsea) [reply content/zh-cn/docs/reference/kubectl/quick-reference.md:43]: The terms like kubelet, kubectl, containerd, and kubeadm are written in all lowercase letters in our conventions. A principle is to follow its official site or logo.

**Pull request merged**